maintenance_tasks.gemspec

```diff
@@ -15,6 +15,7 @@ Gem::Specification.new do |spec|
     "source_code_uri" =>
       "https://github.com/Shopify/maintenance_tasks/tree/v#{spec.version}",
     "allowed_push_host" => "https://rubygems.org",
+    "changelog_uri" => "https://github.com/Shopify/maintenance_tasks/blob/main/CHANGELOG.html",
```

Comment:
Let's copy what Rails and Rubocop are doing and just link to the right release on GitHub instead: https://github.com/rails/rails/blob/fc9ed119c718ea0654e4a5d2477d8391252cbe6b/rails.gemspec#L28, https://github.com/rubocop/rubocop/blob/d8d8474edf63e4b60ec4b72cd78941520f977d44/rubocop.gemspec#L27